stop_words.txt

```diff
@@ -1,3 +1,7 @@
+This file consists of words that the PodSpelling perlcritic policy thinks are incorrect,
+but are spelled correctly (e.g acronyms, words in other languages, etc)
```

Comment:
Commenting follows perl rules, see https://metacpan.org/pod/Pod::Spell 

> The latter is looked at line by line, with anything after an octothorp ("#") removed and then leading and trailing whitespace removed

```suggestion
# This file consists of words that the PodSpelling perlcritic policy thinks are incorrect,
# but are spelled correctly (e.g acronyms, words in other languages, etc)
```

I'm applying this immediatly